testing/apps/tcp/Makefile

```diff
@@ -0,0 +1,18 @@
+APP_NAME 	:= $(shell basename `pwd`)
+IMAGE_NAME 	:= amalgam8/a8-$(APP_NAME)
```

Comment:
If both client and server are in python, why do we need go based build in teh makefile?
All you need is the dockerfiles and a makefile to create those. i.e. you don't need build targets or build flags or any go related stuff.